.github/actions/prepare-cypress/action.yml

```diff
@@ -14,21 +14,21 @@ runs:
   steps:
     # Browsers
     - name: Install Chrome v111
-      if: inputs.is-replay-browser != true
+      if: ${{ inputs.is-replay-browser != 'true' }}
```

Comment:
is it because `is-replay-browser` can't be boolean?

Reply:
It's because GitHub has extremely convoluted and complicated way of matching and casting.
I will document it but for now - this works.